src/edges_cal/alanmode.py

```diff
@@ -425,6 +435,9 @@ def alancal(
     datadir = Path(datadir)
     out = Path(out)
 
+    if s11date is None:
```

Comment:
If `s11date` can be None, it should be given a default in the argument list (and maybe add to the docstring that it will be auto-determined)